tests/unit/Actions.spec.ts

```diff
@@ -15,15 +15,14 @@ import { CorpTypeCd, FilingTypes } from '@/enums'
 import { CourtOrderStepIF, DefineCompanyIF, EffectiveDateTimeIF, IncorporationAgreementIF, NameRequestIF,
   OrgPersonIF, PeopleAndRoleIF, ShareStructureIF, TombstoneIF } from '@/interfaces'
 import { ShareClassIF } from '@bcrs-shared-components/interfaces'
-
-Vue.use(Vuetify)
+import { vi } from 'vitest'
```

Comment:
Jest is replaced by `vi` in Vite. The methods work similar and have similar names as below.

Reply:
Hmm, I've actually haven't stumbled upon the `vitest`.

I've seen that `vi` is the one that's being widely used: https://vitest.dev/api/vi.html

I'm guessing they're the same.

Reply:
OK, I have no further comments on this.